bundles/core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractFieldImpl.java

```diff
@@ -104,6 +104,14 @@ public abstract class AbstractFieldImpl extends AbstractBaseImpl implements Fiel
     @SlingObject
     private Resource resource;
 
+    @ValueMapValue(injectionStrategy = InjectionStrategy.OPTIONAL)
+    @javax.annotation.Nullable
+    protected boolean dorExclusion;
+
+    @ValueMapValue(injectionStrategy = InjectionStrategy.OPTIONAL)
```

Comment:
as discussed today lets keep DoR specific additional props in custom properties.

Reply:
Sure